extension-androidauto/CHANGELOG.md

```diff
@@ -16,6 +17,10 @@ Mapbox welcomes participation and contributions from everyone.
 * Add `MapboxCarMapObserver#onStableAreaChanged` to support all the available functions from the SurfaceCallback. ([#1670](https://github.com/mapbox/mapbox-maps-android/pull/1648))
 * Add support for intercepting the `SurfaceCallback#onClick` when using `MapboxCarMap.prepareSurfaceCallback`. ([#1683](https://github.com/mapbox/mapbox-maps-android/pull/1683))
 
+## Dependencies
+* Android Auto car library `androidx.car.app:app:1.1.+`
+* Supports any version of Mapbox Maps 10.+
```

Comment:
Should we state it here? Since we already have support table at https://github.com/mapbox/mapbox-maps-android/tree/main/extension-androidauto#compatibility-with-maps-sdk-v10 and part of the features require 10.4+

Reply:
Yeah it would be good to link to that compatibility table though

Reply:
There is also this issue https://github.com/mapbox/mapbox-maps-android-internal/issues/1114

We need to remove the `compileOnly` dependency because it is not optional. Making it a hard dependency that can be upgraded.

Reply:
Also just reordered that compatibility list so the newest version is on top

Reply:
https://github.com/mapbox/mapbox-maps-android/pull/1706

Yeah this is expected, having compileOnly allowed us to add incompatible versions into the SDK anyways

Reply:
This bumped the 0.3.0 version to `10.5.0` https://github.com/mapbox/mapbox-maps-android/pull/1276

The `0.3.0` release will always be that way, but we can correct if for the next release.

 There are downsides to supporting old versions by the way, we can't adopt latest apis. I think we should consider adopting latest stable, always.